spec/integration/lib/auth0/api/v2/api_email_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Emails do
+  before(:all) do
+    client = Auth0Client.new(v2_creds)
+    begin
+      client.delete_provider
+    rescue
+      puts 'no email provider to delete'
+    end
+  end
+
+  let(:client) { Auth0Client.new(v2_creds) }
+  let(:name) { 'mandrill' }
+  let(:enabled) { true }
+  let(:credentials) { { 'api_key' => 'api_key' } }
+  let(:settings) { { 'first_setting' =>  'first_setting_set', 'second_setting' => 'second_setting_set' } }
```

Comment:
Because of the comparison in the line 28. The retrieved JSON contains string instead of symbols.